### DISCUSSION TITLE
displays: how to deactivating splash screen on reboot

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#21129] [Q&A] [answered]
i flashed a sunton 4.3" board with 13.4.0.3 (60666fa-lvgl-haspmota) and used Autoconfig for Sonton 5".
all works fine. i wish to daeactivate the splash screen on boot.

I saw tasmota source code has some `#ifdef SHOW_SPLASH`. Does that mean, that the splash is hard coded in this build?

i tried to comment out the `:S`parameter in `display.ini`,no effects.
i did `remove-autoconf` via web gui, but nothing happend.

Before i used `remove-autoconf`, i removed the filepackage that comes with `autoconf`.
The `display.ini` is the one that comes with autoconf

**Comment by sfromis** [comment, marked as answer]
`SetOption135`

**Comment by ChrisVonFrosta** [comment]
really?? I searched several times on command site using strg+f ! Sorry and thank you!
